bottlecap/src/tags/lambda/tags.rs

```diff
@@ -0,0 +1,162 @@
+use std::collections::hash_map;
+use std::sync::Arc;
+
+use crate::config;
+
+// Environment variables for Unified Service Tagging
+const ENV_ENV_VAR: &str = "DD_ENV";
+const VERSION_ENV_VAR: &str = "DD_VERSION";
+const SERVICE_ENV_VAR: &str = "DD_SERVICE";
+
+// Environment variables for the Lambda execution environment info
+const QUALIFIER_ENV_VAR: &str = "AWS_LAMBDA_FUNCTION_VERSION";
+// TODO(astuyve): set runtime
+// const RUNTIME_VAR: &str = "AWS_EXECUTION_ENV";
+const MEMORY_SIZE_VAR: &str = "AWS_LAMBDA_FUNCTION_MEMORY_SIZE";
+const INIT_TYPE: &str = "AWS_LAMBDA_INITIALIZATION_TYPE";
+const INIT_TYPE_KEY: &str = "init_type";
+
+// FunctionARNKey is the tag key for a function's arn
+const FUNCTION_ARN_KEY: &str = "function_arn";
+// FunctionNameKey is the tag key for a function's name
+const FUNCTION_NAME_KEY: &str = "functionname";
```

Comment:
Should we add `AWS_LAMBDA_FUNCTION_NAME` as a tag here too? or we don't care about it in this module?

Reply:
we [don't care about that](https://github.com/DataDog/datadog-agent/blob/main/pkg/serverless/tags/tags.go#L242)